### ISSUE TITLE
Add [Brian Buchanan Buke11@bellsouth.net] #memo-required GDSET2…H32BQC

### ISSUE THREAD

**Issue by stellar-expert-deploy** [stellar-expert/public-directory#232]
Address: `GDSET25DYAJICNTHG56L6I4E4PDAEM7U4H2P6PXXE3ED24SSF4H32BQC`  
Tags: `memo-required`   
Title: **Brian Buchanan**  Domain: Buke11@bellsouth.net  
Requested by @buke78

> Hi I’m Brian   I purchased Xlm tokens  I’m very new to the whole crypto Arena. So basically what happened was I thought I had everything set up in a StellarWallet keys and password when it gave me an address I didn’t realize I had to activate it because I thought it needed the crypto sent to it to activate it. So I bought the Xlm and sent to that address I opened the wallet and it was showing a different address. So of course I contacted the StellarWallet group. We worked together but it came out the wallet and address wasn’t completely activated. I have all the information from buying it to seeing on the stellar explorer. I don’t know what to do. Please help me this is my order.

**Comment by CryptoHobbyist** [comment]
@buke78 
When you say "StellarWallet" what web address were you setting this up at? NEVER give anyone your private key for any reason. Many times scammers act like they are part of support but once they have your private key, your stuff is gone.

**Comment by buke78** [comment]
It was a caramba wallet app. The address that I used was from them but it wasn’t active. I talked to them they said they couldn’t find it. But my transaction is just sitting there on the stellar explorer. Yeah I heard the scammers are getting bad.

**Comment by CryptoHobbyist** [comment]
Did you look up the transaction and your wallet on stellar.expert?

**Comment by CryptoHobbyist** [comment]
How did you come upon them as a wallet to use?

**Comment by buke78** [comment]
I have the copy of the transactions when I did plus the transaction e-mail copy of that and yes I found it on the stellar expert. And I made a copy of that to matching the address and the amount of Xlm. Thank you for taking your time out to help me.

**Comment by CryptoHobbyist** [comment]
@buke78 Ask more if you have questions.

**Comment by buke78** [comment]
What do I need to do now any advice from here? Thank you for everything so far.

**Comment by CryptoHobbyist** [comment]
@buke78 Brian sorry I missed your last comments. Closed just meant the it was believed to be resolved (as far as what Stellar.expert can do. 

"I have the copy of the transactions when I did plus the transaction e-mail copy of that and yes I found it on the stellar expert. And I made a copy of that to matching the address and the amount of Xlm. Thank you for taking your time out to help me."

I thought that you might have gotten things resolved with caramba wallet app. They would have to be the ones to help you with funds in/transferred to their wallet (I do not know them and cannot speak to their legitimacy). But if funds were sent or arranged with their system then you would have to have them help you. If you post the Stellar transaction number (never give secret keys to anyone) I could also look.
